packages/admin-next/dashboard/src/routes/orders/order-create-claim/components/claim-create-form/claim-create-form.tsx

```diff
@@ -161,14 +166,18 @@ export const ClaimCreateForm = ({
       )
 
       return Promise.resolve({
-        inbound_items: previewItems.map((i) => ({
-          item_id: i.id,
-          quantity: i.detail.return_requested_quantity,
-          note: i.actions?.find((a) => a.action === "RETURN_ITEM")
-            ?.internal_note,
-          reason_id: i.actions?.find((a) => a.action === "RETURN_ITEM")?.details
-            ?.reason_id,
-        })),
+        inbound_items: previewItems.map((i) => {
+          const returnAction = i.actions?.find(
```

Comment:
thought: Should we check if we found a `returnAction` here before sending it to the backend, or does it not matter?

Reply:
The return action will always be present here. Right now in the previewItems array, there are only inbound items as we can only create those atm. When I get to creating other types of actions, I will have to change the logic of the previewItems here and extract inboundItems from there. At that point, this logic here will be removed.

Reply:
Got it, thx 👍